packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -248,12 +248,29 @@ pub struct LinkTypeRootedSubgraph {
     pub link_type: PersistedLinkType,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, ToSchema)]
+pub struct PersistedOntologyMetadata {
+    pub identifier: PersistedOntologyIdentifier,
```

Comment:
Why have you made this pub?

Reply:
This was a mistake, addressed in https://github.com/hashintel/hash/pull/1199/commits/396765840d31f7cc5dc9af2feb3315e0aecfdc82